snuba/subscriptions/executor_consumer.py

```diff
@@ -106,6 +107,22 @@ def get_topics_for_entity(
         strict_offset_reset=strict_offset_reset,
     )
 
+    try:
+        partition_count = get_partition_count_from_kafka_admin(
+            scheduled_topic_spec.topic, logger
+        )
+    except Exception:
```

Comment:
What exception is this meant to be catching? That function is never expected to raise an error

Reply:
If we exceed the number of attempts we re raise the error

Reply:
Ah ok, didn't see that earlier